src/Pillar-ExporterCore.package/PRTocBuilderVisitor.class/instance/visitInputFileAnnotation..st

```diff
@@ -6,7 +6,7 @@ visitInputFileAnnotation: anAnnotation
 	currentSave := self currentFile.
 	newSource := self documentSource parent resolve: anAnnotation path.
 	self validateInclusionOfFile: newSource.
-	document := PRPillarParser parse: newSource contents.
+	document := PRPillarParser new optimize parse: newSource contents.
```

Comment:
This one should be

`PRPillarParserMain parse:`

This is not Juraj's fault, this was like since before. But the idea is that PRPillarParserMain is the entry point where the current used parser is just registered.

Reply:
changed